### PR TITLE
Update copy on OpenStack managed page

### LINKED ISSUE #3068: Joule logo is blurry
On the /download/iot page. The logo of Intel Joule is blurry:
screenshot-2018-4-26 download ubuntu for iot boards download ubuntu

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Updated the copy on /openstack/managed

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page has been updated according to [the copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#) and resolve the comments

## Issue / Card

Fixes #3068

### CODE REVIEW

File: templates/openstack/managed.html [hunk 1]
```diff
@@ -45,9 +46,8 @@ <h1>Fully managed OpenStack private cloud. Half the cost of VMware.</h1>
...
-      <p>"Hosted private cloud infrastructure: a cost analysis"</p>
-      <a href="https://www.brighttalk.com/webcast/6793/387966?utm_source=brighttalk-portal&utm_medium=web&utm_content=hosted%20private%20cloud%20infrastructure&utm_campaign=webcasts-search-results-feed" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch the webinar', 'eventLabel' : 'Managed OpenStack webinar - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Watch the webinar</a>
-      <a href="/openstack/contact-us?product=cloud-managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Bootstack - Top CTA', 'eventValue' : undefined });" class="p-button--neutral js-invoke-modal">Talk to us</a>
+      <a href="/openstack/contact-us?product=cloud-managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Bootstack - Top CTA', 'eventValue' : undefined });" class="p-button--neutral js-invoke-modal is-inline">Talk to us</a>
```
Comment (pmahnke) [inline templates/openstack/managed.html:49]:
```suggestion
      <a href="/openstack/contact-us?product=cloud-managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Bootstack - Top CTA', 'eventValue' : undefined });" class="p-button--positive u-no-margin--left js-invoke-modal is-inline">Talk to us</a>
```

File: templates/openstack/managed.html [hunk 2]
```diff
@@ -45,9 +46,8 @@ <h1>Fully managed OpenStack private cloud. Half the cost of VMware.</h1>
...
+      <a href="https://www.brighttalk.com/webcast/6793/387966?utm_source=brighttalk-portal&utm_medium=web&utm_content=hosted%20private%20cloud%20infrastructure&utm_campaign=webcasts-search-results-feed" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch the webinar', 'eventLabel' : 'Managed OpenStack webinar - Top CTA', 'eventValue' : undefined });">Watch the "Hosted private cloud infrastructure: a cost analysis" webinar</a>
```
Comment (pmahnke) [inline templates/openstack/managed.html:50]:
```suggestion
      <a href="https://www.brighttalk.com/webcast/6793/387966?utm_source=brighttalk-portal&utm_medium=web&utm_content=hosted%20private%20cloud%20infrastructure&utm_campaign=webcasts-search-results-feed" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch the webinar', 'eventLabel' : 'Managed OpenStack webinar - Top CTA', 'eventValue' : undefined });">Watch the "Hosted private cloud infrastructure: a cost analysis" webinar&nbsp;&rsaquo;</a>
```

**Review by pmahnke — APPROVED** [review]
+1 with changes - also, you missed the top of the copy doc and there was an old error I fixed

File: templates/openstack/managed.html (hunk 1 above)
Comment (pmahnke) [reply templates/openstack/managed.html]: @steverydz did you except this?

File: templates/openstack/managed.html (hunk 2 above)
Comment (pmahnke) [reply templates/openstack/managed.html]: @steverydz did you except this?

File: templates/openstack/managed.html (hunk 1 above)
Comment (steverydz) [reply templates/openstack/managed.html]: yes, it's in the changes

**Comment by pmahnke** [comment]
cool, it was hard to follow

**Pull request merged**